### PR TITLE
Bump opset version to 20

### PR DESCRIPTION
**Pull request by yuanyao-nv**
### Description
Release branch rel-1.14.0 has been cut. Bump opset number to 20.

### CODE REVIEW

File: VERSION_NUMBER [hunk 1]
```diff
@@ -1 +1 @@
-1.14.0
+1.14.1
```
Comment (justinchuby) [inline VERSION_NUMBER:1]: To 1.15.0?

Comment (yuanyao-nv) [reply VERSION_NUMBER]: Updated. Do we no plan to have a minor release after this?

Comment (justinchuby) [reply VERSION_NUMBER]: I think we can just cut rel branches and change this for minor releases. That's what pytorch does.

**Review by jcwchen — APPROVED** [review]
LGTM. Thanks!

**Pull request merged**